### ISSUE TITLE
Allow partials to be accessed from subfolders in root

### ISSUE THREAD

**Issue by mariocontext** [vapid/vapid#97]
Right now, partials need to be in the root www folder in order to be accessed. It would be nice to be able put into subfolders so a more component based approach can be applied for better organization. Ran this originally by Scott who said to create a feature request.

In Story format: As a Vapid template developer, when I am creating pieces of reusable components, I want to be able to put them in subfolders, so that the components are well organized.

**Comment by Hadr619** [comment]
Having been working with Vapid on a larger project this could be very beneficial for the project hierarchy.

**Comment by heyflo** [comment]
Indeed, that's one of the first things I have tried and it had to be in the root `www` folder.

**Comment by srobbin** [comment]
This just shipped in the latest release. Thanks for feedback.

As a reminder, the NPM package changed from `vapid-cli` to `@vapid/cli`. To upgrade:

```
npm uninstall -g vapid-cli
npm install -g @vapid/cli
```
